Automattic/wp-calypso
# Pattern Library: Fix double `calypso_pattern_library_view` call

Fixes https://github.com/Automattic/dotcom-forge/issues/6222

## Proposed Changes

If user settings are slow to load, that can sometimes cause double `calypso_pattern_library_view` events to be submitted. This PR fixes that.

## Testing Instructions



1. Navigate to `/patterns`
2. Ensure that a single `calypso_pattern_library_view` Tracks event is submitted and the `user_is_dev_account` value is correct for your account
